src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -142,7 +142,12 @@ public static class ConfigurationKeys
         /// <summary>
         /// Configuration key for setting the size of the trace buffer
```

Comment:
```suggestion
        /// Configuration key for setting the size of the trace buffer in bytes
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1151/commits/036ae23d5ace2c8e19264ac624a2c0ec63de27e6